src/main/java/com/blackducksoftware/integration/hub/dataservice/model/ComplexLicenseModel.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Hub Common
+ *
+ * Copyright (C) 2017 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.hub.dataservice.model;
+
+import java.util.List;
+
+import org.apache.commons.lang3.builder.RecursiveToStringStyle;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import com.blackducksoftware.integration.hub.api.component.version.ComplexLicenseItem;
+
+public class ComplexLicenseModel {
```

Comment:
If this class has MetaInformation then it should extend HubItem and should be called an 'Item' not a 'Model'
If it is indeed a model around HubItems then this object should not have MetaInformation to deal with